dcat/index.html

```diff
@@ -4635,6 +4635,13 @@ <h3>Compressed and packaged distributions</h3>
 <section id="changes" class="appendix">
     <h2>Change history</h2>
     <p>A full change-log is available on <a href="https://github.com/w3c/dxwg/commits/gh-pages/dcat">GitHub</a></p>
+    <section id="changes-since-20140116">
+        <h3>Changes since the W3C Recommendation of 4 February 2020</h3>
+        <p>The document has undergone the following changes since the W3C Recommendation of 4 February 2020 [[?VOCAB-DCAT-2-20200204]]:</p>
```

Comment:
@riccardoAlbertoni I would recommend referring to the version number in additon to including the date; to be consistent we could also do that with the first recommendation, which is now DCAT1

Reply:
@agbeltran my last two commits add the version number. DCAT 1 is called DCAT 2014 in the spec. Do you want me to replace all the "DCAT 2014" with "DCAT 1" ?

Reply:
thanks @riccardoAlbertoni ! I think that up until now we've referred to DCAT 2014, but it would make sense to start calling it DCAT 1 - what do you think?

Reply:
I think it makes sense..having versions called differently is not ideal. I am going to prepare an update that implements this change.

Reply:
thanks @riccardoAlbertoni !